docs/_automation/waste-reduction-logic-app/README.md

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Don't forget to add the header needed for Just the Docs. Look at other doc files for an example.